src/GroupTypeManager.php

```diff
@@ -227,9 +227,22 @@ public function getGroupsForEntityType($entity_type_id) {
    *   An associative array, keyed by entity type, each value an indexed array
    *   of bundle IDs.
    */
-  public function getAllGroupBundles($entity_type = NULL) {
+  public function getAllGroupBundles() {
+    return $this->getGroupMap();
+  }
```

Comment:
This doesn't make sense to me, why have a method that doesn't do anything but wrap another method?

Reply:
I'll add todo for that one.